### PR TITLE
Convert to parent-pom

### PR DESCRIPTION
**Pull request by kwhetstone**
Converted to parent pom to avoid test failures due to upgraded versions of htmlunit in core.  There are still test failures with junit, but those are unrelated to the parent pom.

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
Please do not forget @reviewbybees 
:bee: anyway

File: pom.xml [hunk 1]
```diff
@@ -17,6 +17,10 @@
             <url>http://opensource.org/licenses/MIT</url>
         </license>
     </licenses>
+    <properties>
+        <jenkins.version>1.580.1</jenkins.version>
+        <jenkins-test-harness.version>1.580.1</jenkins-test-harness.version>
```
Comment (andresrc) [inline pom.xml]: Have you tried compiling against JTH 2.1 (the default)?

Comment (jglick) [inline pom.xml]: Yeah prefer use of the current test harness.

Comment (kwhetstone) [inline pom.xml]: If I do that, I have to update the junit test classes to the new htmlunit.  It won't compile if I use 2.1.  I could make those changes here, but I was thinking it would be handled in a different pull request.

Comment (andresrc) [inline pom.xml]: In other plugins conversions the same pull request has included the,  update the parent POM,  test fixes so that they compile with JTH 2.1 and javadoc fixes so that it does the build does not break on Java 8 If the changes are not too big, encourages adoption.

Comment (amuniz) [inline pom.xml]: +1 to update JTH

**Comment by jglick** [comment]
Superseded I think.
